src/main/java/xt9/deepmoblearning/common/inventory/ContainerExtractionChamber.java

```diff
@@ -144,7 +145,12 @@ public ItemStack transferStackInSlot(EntityPlayer player, int index) {
     }
 
     @Override
-    public boolean canInteractWith(EntityPlayer player) {
-        return !player.isSpectator();
```

Comment:
Why was this check removed?

Reply:
I do not see the point in it, but you can bring it back.